site/content/docs/monitoring-as-code/getting-started.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Getting started
+weight: 1
+menu:
+  docs:
+    parent: "Monitoring-as-Code"
+---
+
+The Checkly Terraform provider enables you to declare your monitoring setup as code using [HashiCorp Terraform](https://www.terraform.io/). You can get started with it in a matter of minutes by following the steps shown below.
+
+{{<info>}}
+For in-depth information on Terraform, please see HashiCorp's [official documentation](https://registry.terraform.io/providers/checkly/checkly/latest/docs).
+{{</info>}}
+
+## Installation
+
+To get started, you need to install the [latest version](https://www.terraform.io/downloads) of Terraform.
+
+Next, switch to a new folder for your project, and create your `main.tf` file:
+
+```terraform
+terraform {
+  required_providers {
+    checkly = {
+      source = "checkly/checkly"
+      version = "~> 1.0"
+    }
+  }
+}
+
+variable "checkly_api_key" {}
+variable "checkly_account_id" {}
+
+provider "checkly" {
+  api_key = var.checkly_api_key
+  account_id = var.checkly_account_id
+}
+```
+
+To use the provider with your Checkly account, you will need an API Key for your Checkly user. Go to the [API keys tab](https://app.checklyhq.com/settings/user/api-keys) in your user settings and click `Create API key`. Get your User API key and add it to your env:
+
+```bash
+$ export TF_VAR_checkly_api_key=cu_xxx
+```
+
+You also need to set your target account ID, which you can find under your [account settings](https://app.checklyhq.com/settings/account/general). If you don't have access to account settings, please contact your account owner/admin.
+
+```bash
+$ export TF_VAR_checkly_account_id=xxx
+```
+
+{{<info>}}
+If you do not export these environment variables, Terraform will ask you to enter them later.
+{{</info>}}
+
+Running `terraform init` will install the Checkly Terraform provider for you, as well as initialising your project. You can now start adding resources to your file. You can check the official documentation to see [available parameters](https://registry.terraform.io/providers/checkly/checkly/latest/docs/resources/check) for each resource type. As an example, you could add a basic browser check resource at the bottom of your `main.tf`:
```

Comment:
Where do I run terraform init in my terminal?

Reply:
https://registry.terraform.io/providers/checkly/checkly/latest/docs/resources/check
Is that link relevant knowing that you are describing all resources in the docs?

Reply:
@hlenke I have specified that this happens in the terminal.
The link is still relevant because I am not describing every single parameter in our docs - the definitive guide containing every option is and will remain on the registry.